master/internal/command/command.go

```diff
@@ -481,6 +481,16 @@ func (c *command) Receive(ctx *actor.Context) error {
 		}
 		ctx.Respond(&apiv1.SetTensorboardPriorityResponse{Tensorboard: c.toTensorboard(ctx)})
 
+	case *apiv1.DeleteWorkspaceRequest:
+		if c.Metadata.WorkspaceID != model.AccessScopeID(msg.Id) {
+			// on workspace mismatch we silently ignore the request as these are sent out in batch.
+			return nil
+		}
+		ctx.Tell(c.allocation, sproto.AllocationSignalWithReason{
+			AllocationSignal:    sproto.KillAllocation,
+			InformationalReason: "user requested workspace delete",
+		})
```

Comment:
I think you'll have to respond to this message, otherwise the actor will be blocked. You'll have to add this case to shells, notebooks, and tensorboard actors too.

Reply:
we chatted a bit offline.

leaving this here so I don't forget
re failing tests: https://github.com/determined-ai/determined/pull/5715/commits/687ff93beb2b65c1f6c4e3f24277e7365ca99acb was the cause and not lack of an explicit response to the msgs

> it looks like the actor implementation is fine with Asks not being responded to.
https://github.com/hamidzr/determined/blob/28c7b895dac248b81743c6a4d1544ba03135e9b8/master/pkg/actor/response.go#L14-L18 I expect that this still waits for the actor to Receive this msg before returning.